src/util/expr_util.h

```diff
@@ -72,4 +72,7 @@ bool has_subtype(const typet &, const irep_idt &id, const namespacet &);
 /// lift up an if_exprt one level
 if_exprt lift_if(const exprt &, std::size_t operand_number);
 
+/// returns true if the expression can be used as an lvalue
```

Comment:
As much as I hate having to add the comment, the coding standards say docs should be on the implementation.